.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+blank_issues_enabled: false
+contact_links:
+  - name: Eclipse JKube Community Chat
+    url: https://gitter.im/eclipse/jkube
+    about: Please ask and answer questions here.
+  - name: Eclipse JKube Mailing List
+    url: https://accounts.eclipse.org/mailing-list/jkube-dev
+    about: Eclipse JKube Developer Discussions
```

Comment:
I'm not sure how these get displayed.
We've got at  least 4 communication channels that we might want to encourage
- Gitter
  https://gitter.im/eclipse/jkube
- Mailing List
  https://accounts.eclipse.org/mailing-list/jkube-dev
- GH Discussions
  https://github.com/eclipse/jkube/discussions
- Stack Overflow
  https://stackoverflow.com/questions/ask?tags=jkube

I'd recommend Gitter only for short, simple questions, and for casual interaction with the maintainers.

I don't know how I'd classify the others, ideas?

WDYT?

Reply:
I was expecting some kind of feedback here TBH

Reply:
It looked good to me. I had updated `config.yml` as per suggestion